app/renderer/ui/components/main/sections/dataRequests/tabs/editor/index.tsx

```diff
@@ -1,41 +1,140 @@
+import CardDefault from "app/renderer/ui/components/card/default"
 import * as React from "react"
 
+import { TabInfo, TabComponent } from "app/renderer/ui/components/main/sections"
 import * as urls from "app/renderer/constants/urls"
-import { TabComponent, TabInfo } from "app/renderer/ui/components/main/sections"
+import Wrapper from "app/renderer/ui/components/wrapper"
 import { Services } from "app/renderer/services"
 
+const svgBar = require("svg/dataRequestsEditorBar.svg")
+const svgRetrieval = require("svg/dataRequestsEditorRetrieval.svg")
+const svgAttestation = require("svg/dataRequestsEditorAttestation.svg")
+const svgDelivery = require("svg/dataRequestsEditorDelivery.svg")
+
+const styles = require("./style.scss")
+const tabStyles = require("app/renderer/ui/components/main/sections/smartContracts/tabs/style.scss")
+
 /**
  * Props that contain configuration
  *
  * @interface Props
  */
 interface Props {
-  services: Services,
+  services: Services
 }
-
 /**
- * TabEditor component
+ * Smart Contracts Editor tab component
  *
  * @class TabEditor
- * @extends {TabComponent<any & Props>}
+ * @extends {TabComponent<any>}
  */
 class TabEditor extends TabComponent<any & Props> {
-
   // tslint:disable-next-line:prefer-function-over-method completed-docs
   public render() {
     return (
-      <div>
-        Data requests section - editor tab
+      <div className={styles.layout}>
+        <div className={`${styles.bar}`}>
+          <img src={svgBar} />
+        </div>
+        <div className={`${tabStyles.left} ${styles.left}`}>
+          <Wrapper
+            title="RETRIEVAL LOGIC"
+          >
+            <CardDefault
+              className={styles.editor}
+            >
+              <img src={svgRetrieval} />
+            </CardDefault>
+          </Wrapper>
+          <br />
+          <Wrapper
+            title="ATTESTATION LOGIC"
+          >
+            <CardDefault
+              className={styles.editor}
+            >
+              <img src={svgAttestation} />
+            </CardDefault>
+          </Wrapper>
+          <br />
+          <Wrapper
+            title="DELIVERY LOGIC"
+          >
+            <CardDefault
+              className={styles.editor}
+            >
+              <img src={svgDelivery} />
+            </CardDefault>
+          </Wrapper>
+        </div>
+        <div className={`${tabStyles.right} ${styles.right}`}>
+          <div className={styles.right}>
+            <p className={styles.title}>FILE NAME</p>
+            <p className={styles.text}>ethereum-weather-reporter.rad</p>
+
+            <hr />
```

Comment:
Sorry, I should have explained this to you. These are actually input fields. The horizontal lines are not separators but the actual bottom line of the input fields.